net-analyzer/nagios-plugins-ceph/nagios-plugins-ceph-1.5.0.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit python-single-r1
+
+DESCRIPTION="Nagios plugins for Ceph"
+HOMEPAGE="https://github.com/ceph/ceph-nagios-plugins"
+SRC_URI="https://github.com/ceph/ceph-nagios-plugins/archive/ceph-nagios-plugins_${PV}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+
+DEPEND="
```

Comment:
If you don't need to call `fowners`/`fperms` later on, these can be moved to `RDEPEND`.